recipe/conda_build_config.yaml

```diff
@@ -47,12 +59,16 @@ cl_version:
 # curl -ILSs https://aka.ms/vs/17/release/vc_redist.x64.exe | grep "Location:"
 # curl -ILSs https://aka.ms/vs/17/release/vc_redist.arm64.exe | grep "Location:"
 uuid:
+ - 1754ea58-11a6-44ab-a262-696e194ce543
+ - 1754ea58-11a6-44ab-a262-696e194ce543
  - 1754ea58-11a6-44ab-a262-696e194ce543
  - 1754ea58-11a6-44ab-a262-696e194ce543
  - 71c6392f-8df5-4b61-8d50-dba6a525fb9d
  - c7707d68-d6ce-4479-973e-e2a3dc4341fe
  - b929b7fe-5c89-4553-9abe-6324631dcc3a
 sha256:
+ - 15B8F5B2106DC7A7BD83AB57B796770E0F4ECB891AD19BF655C9D6A9DA650AD2
+ - 3642E3F95D50CC193E4B5A0B0FFBF7FE2C08801517758B4C8AEB7105A091208A
  - 15B8F5B2106DC7A7BD83AB57B796770E0F4ECB891AD19BF655C9D6A9DA650AD2
  - 3642E3F95D50CC193E4B5A0B0FFBF7FE2C08801517758B4C8AEB7105A091208A
```

Comment:
I had begun preparing a PR for this a while ago, but it seemed to me that they haven't updated the `vc_redist` link yet? Pointing to the exact same uuic/sha sounds suspicious, no?

Reply:
> Pointing to the exact same uuic/sha sounds suspicious, no?

Not really. Some compiler updates don't come with runtime updates.

Reply:
OK, fair enough; can't remember such a case in recent memory though. 🤷 

Separately, the GHA/AZP images are [still](https://github.com/actions/runner-images/blob/d33c9005ccc308dfce8ff1710cc1fcbc5c6b22ac/images/windows/Windows2022-Readme.md#visual-studio-enterprise-2022) on 17.10/19.40 AFAICT

Reply:
They briefly updated it (See https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/1010969/logs/24) and reverted it